src/components/ContentNode/LinkableHeading.vue

```diff
@@ -17,10 +17,10 @@
       v-if="shouldLink"
       :to="{ hash: `#${anchor}` }"
       class="header-anchor"
-      aria-label="Scroll to section"
       @click="handleFocusAndScroll(anchor)"
     >
       <slot />
+      <span class="visuallyhidden" >{{ $t('accessibility.in-page-link') }}</span>
```

Comment:
Works great, but I think you'll need to update the unit tests for this change (they're failing now)

Reply:
Oh yeah, of course 🤦‍♀️  https://github.com/apple/swift-docc-render/pull/698/commits/8259023f76d5a62b9ba3ed7b7214fa9cf2756ba0